regression-test/suites/nereids_p0/aggregate/aggregate.groovy

```diff
@@ -141,8 +141,8 @@ suite("aggregate") {
     qt_aggregate """ select variance(c_bigint), variance(distinct c_double) from ${tableName}  """
     qt_aggregate """ select 1 k1, 2 k2, c_bigint k3, sum(c_double) from ${tableName} group by 1, k2, k3 order by k1, k2, k3 """
     qt_aggregate """ select (k1 + k2) * k3 k4 from (select 1 k1, 2 k2, c_bigint k3, sum(c_double) from ${tableName} group by 1, k2, k3) t order by k4 """
-    qt_aggregate32" select topn_weighted(c_string,c_bigint,3) from ${tableName}"
-    qt_aggregate33" select avg_weighted(c_double,c_bigint) from ${tableName};"
+    // qt_aggregate32" select topn_weighted(c_string,c_bigint,3) from ${tableName}"
+    // qt_aggregate33" select avg_weighted(c_double,c_bigint) from ${tableName};"
```

Comment:
avg_weighted do not return array why remove it?

Reply:
by mistake, because topn_weighted return array